runtime/lua/vim/snippet.lua

```diff
@@ -477,7 +477,9 @@ end
 function M.expand(input)
   local snippet = G.parse(input)
   local snippet_text = {}
-  local base_indent = vim.api.nvim_get_current_line():match('^%s*') or ''
+  local curbuf = vim.api.nvim_get_current_buf()
+  local expandtab = vim.bo[curbuf].expandtab
```

Comment:
Nit: To keep the convention of using snake_case variable names
```suggestion
  local expand_tab = vim.bo[curbuf].expandtab
```

Reply:
But isn't that inconsistent with the option name?